src/storybook/stories/Section.stories.tsx

```diff
@@ -0,0 +1,194 @@
+                  label="ButtonSectionItem"
+                  onPress={() => {}}
+                />
+                <ActionSectionItem
+                  text="ActionSectionItem"
+                  onPress={() => {}}
+                />
+                <CounterSectionItem
+                  text="CounterSectionItem"
+                  count={2}
+                  addCount={() => {}}
+                  removeCount={() => {}}
+                />
+                <ExpandableSectionItem
+                  text="ExpandableSectionItem"
+                  onPress={() => {}}
+                  expandContent={<ThemeText>Content</ThemeText>}
+                />
+                <FavoriteDepartureSectionItem
+                  favorite={{
+                    id: '1',
+                    stopId: '2',
+                    lineId: '3',
+                    quayId: '4',
+                    quayName: 'FavoriteDepartureSectionItem',
+                  }}
+                />
+                <FavoriteSectionItem
+                  favorite={{
+                    id: '1',
+                    location: {
+                      id: '2',
+                      name: 'FavoriteSectionItem',
+                      layer: 'address',
+                      coordinates: {longitude: 3, latitude: 4},
+                      locality: '',
+                      category: [],
+                      resultType: 'favorite',
+                      favoriteId: '5',
+                    },
+                  }}
+                />
+                <GenericClickableSectionItem>
+                  <ThemeText>GenericClickableSectionItem</ThemeText>
+                </GenericClickableSectionItem>
+                <GenericSectionItem>
+                  <ThemeText>GenericSectionItem</ThemeText>
+                </GenericSectionItem>
+                <InternalLabeledSectionItem label="InternalLabeledSectionItem">
+                  <ThemeText>Content</ThemeText>
+                </InternalLabeledSectionItem>
+                <LinkSectionItem text="LinkSectionItem" />
+                <LocationInputSectionItem
+                  label="LocationInputSectionItem"
+                  onPress={() => {}}
+                />
+                <MessageSectionItem
+                  messageType="valid"
+                  message="MessageSectionItem"
+                />
+                <PhoneInputSectionItem
+                  label="PhoneInputSectionItem"
+                  prefix="+47"
+                  onChangePrefix={() => {}}
+                />
+                <TextInputSectionItem label="TextInputSectionItem" />
+                <ToggleSectionItem
+                  text="ToggleSectionItem"
+                  onValueChange={() => {}}
+                />
+                <RadioGroupSection
+                  items={['Radio group option 1', 'Radio group option 2']}
+                  keyExtractor={(s) => s}
+                  itemToText={(t) => t}
+                />
```

Comment:
<img width="317" alt="Skjermbilde 2024-02-29 kl  09 36 08" src="https://github.com/AtB-AS/mittatb-app/assets/134292729/6955d8a9-0398-42ea-82f3-f39d165ef6c6">

I'm not sure, is the borderRadius for `RadioGroupSection` meant to be like this?

Reply:
I didn't make any changes to it, so this is how it will currently appear if it is added inside a Section alongside other section items. As there will be a separate task for adding `RadioGroupSection` to Storybook, this might be something to follow up in that task is my suggestion.

Reply:
Added it as it's own decorator, and where it's used in the code I removed the `<Section>` wrapping of it :)